### PR TITLE
`gpmpn-dropdown-navigation.js`: Added snippet for displaying multi page navigation as a dropdown.

### PR DESCRIPTION
**Pull request by saifsultanc**
## Context

⛑️ Ticket(s): https://secure.helpscout.net/conversation/2660745826/69190?folderId=7098280

## Summary

Created a snippet that creates a drop down based on the current pages. Pages with actual page links from GPMPN will be enabled. Pages without page links will be disabled. Select a page will trigger a click on the corresponding page link, navigating the form to that page.

**Demo:**
https://www.loom.com/share/38fcc941036842d6a81a599fd8de39d7

### CODE REVIEW

File: gp-multi-page-navigation/gpmpn-dropdown-navigation.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			// page not yet traversed, should be disabled
+			$option.attr( 'disabled', 'disabled' );
+		}
+
+		$select.append( $option );
+	} );
+}
+
+// Initialize the navigation state on page load
+setTimeout( updateNavigationState, 200 );
```
Comment (spivurno) [inline gp-multi-page-navigation/gpmpn-dropdown-navigation.js:58]: I didn't need the 200 in my testing. What issue did you see?

Comment (spivurno) [reply gp-multi-page-navigation/gpmpn-dropdown-navigation.js:58]: Sorry, @saifsultanc, I meant keeping the setTimeout but removing the 200. 

```
setTimeout( updateNavigationState );
```

Comment (saifsultanc) [reply gp-multi-page-navigation/gpmpn-dropdown-navigation.js:58]: @spivurno Okay yeah. We can do that. That did work. But, I was being extra cautious with the page render. Will push the update to the PR.

**Review by spivurno — APPROVED** [review]
LGTM!

**Pull request merged**